### ISSUE TITLE
Job 1, “kcachegrind callgrind.out.filename…” terminated by signal SIGSEGV (Address boundary error)

### ISSUE THREAD

**Issue by akalyaev** [virtan/eep#10]
Is it a kcachegrind issue?

```
λ uname -a
Linux 3.13.0-85-generic #129-Ubuntu SMP x86_64 x86_64 x86_64 GNU/Linux
λ kcachegrind -v
Qt: 4.8.6
KDE Development Platform: 4.13.3
KCachegrind: 0.7.4kde
```

**Comment by virtan** [comment]
Please, try running kcachegrind with no arguments and open callgrind.out.filename from UI.
The second idea is to cut the size of callgrind.out file and try open a shorter file.

**Comment by akalyaev** [comment]
Thank you for the tips!

Updating ubuntu from 14.04 to 16.04 fixed the bug (or maybe it was Qt `4.8.6` versus `4.8.7`, not exactly sure).

```
λ kcachegrind -v
Qt: 4.8.7
KDE Development Platform: 4.14.16
KCachegrind: 0.7.4kde
λ uname -a
Linux 4.4.0-22-generic #40-Ubuntu SMP x86_64 x86_64 x86_64 GNU/Linux
```
